src/lib/core/components/visualizations/implementations/BarplotVisualization.tsx

```diff
@@ -301,47 +337,60 @@ function BarplotViz(props: Props) {
             marginTop: 20,
           }}
           showSpinner={data.pending}
+          dependentAxisLogScale={vizConfig.dependentAxisLogScale}
         />
       )}
     </div>
   );
 }
 
 type BarplotWithControlsProps = BarplotProps & {
-  vizType?: string;
+  dependentAxisLogScale: boolean;
+  onDependentAxisLogScaleChange: (newState: boolean) => void;
+  valueSpec: ValueSpec;
+  onValueSpecChange: (newValueSpec: ValueSpec) => void;
 };
 
 function BarplotWithControls({
   data,
-  vizType,
-  ...BarplotControlsProps
-}: // eslint-disable-next-line @typescript-eslint/no-unused-vars
-BarplotWithControlsProps) {
-  // TODO Use UIState
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const errorManagement = useMemo((): ErrorManagement => {
-    return {
-      errors: [],
-      addError: (_: Error) => {},
-      removeError: (_: Error) => {},
-      clearAllErrors: () => {},
-    };
-  }, []);
-
+  dependentAxisLogScale,
+  onDependentAxisLogScaleChange,
+  valueSpec,
+  onValueSpecChange,
+  ...barPlotProps
+}: BarplotWithControlsProps) {
   return (
     <div style={{ display: 'flex', flexDirection: 'column' }}>
       <Barplot
-        {...BarplotControlsProps}
+        {...barPlotProps}
+        dependentAxisLogScale={dependentAxisLogScale}
         data={data}
         // add controls
         // displayLegend={true}
         displayLibraryControls={false}
       />
-      {/* potential BarplotControls: commented out for now  */}
-      {/* <BarplotControls
-          label="Bar Plot Controls"
-          errorManagement={errorManagement}
-        /> */}
+      <div style={{ display: 'flex', flexDirection: 'row' }}>
+        <LabelledGroup label="Y-axis">
+          <Switch
+            label="Log Scale:"
+            state={dependentAxisLogScale}
+            onStateChange={onDependentAxisLogScaleChange}
+          />
+          <RadioButtonGroup
+            selectedOption={
+              valueSpec === 'proportion' ? 'proportional' : 'count'
+            }
+            options={['count', 'proportional']}
+            onOptionSelected={(newOption) => {
+              if (newOption === 'proportional') {
```

Comment:
Some of this EDA code might be simplified if the RadioButtonGroup component took two props: `displayLabels` and `values`.